### DISCUSSION TITLE
Script times out on simple codes

### DISCUSSION THREAD

**Discussion by longj08** [sadieblue/agilityblue.workspace-scripts#39] [Q&A] [answered]
Hi, 

I am trying to get a billing entry record with the below codes.  When I ran it on the testing environment workspace, it timed out but when I ran it on the live environment (with a different id number) workspace, I got the answer right away.  Do you know what is causing the issues on the testing environment workspace?  

$billingEntry = get-BillingEntry -id 204
write-output "BillingEntry:"
$billingEntry | ConvertTo-json -depth 10

Thanks,
Jack

**Comment by mrabb24** [comment, marked as answer]
Hi Jack,

Timeout issues can happen for a variety of reasons such as network congestion, database locks, and application errors. I don't see anything wrong with your script syntactically-speaking, so If you're still experiencing this issue, I'd recommend that you contact support@sadiebluesoftware.com so the correct people can look at server logs and help you troubleshoot the issue further.

Matt
